skins/laika/src/store/util.ts

```diff
@@ -11,12 +12,21 @@ export const action = buildActionOrMutationName;
 export const mutation = buildActionOrMutationName;
 
 export const Helper = {
-	inputIsValid: ( value: string, pattern: string ) => {
+	inputIsValid: ( value: string, pattern: string, isOptional?: boolean ) => {
+		if ( isOptional && Helper.isEmpty( value ) ) {
+			return Validity.VALID;
+		}
 		if ( pattern === null ) {
 			return value !== '' ? Validity.VALID : Validity.INVALID;
 		}
+		if ( Helper.isEmpty( value ) ) {
```

Comment:
uhm, isn't this just `isEmpty( value )`

Reply:
Typescript.

Reply:
then it must be `this.isEmpty`, referencing its own name is just weird :D